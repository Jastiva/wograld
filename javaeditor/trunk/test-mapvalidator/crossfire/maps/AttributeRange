arch map
name test for AttributeRange map validator
difficulty 1
width 6
height 2
msg
Created:  2010-06-24 Andreas Kirschbaum
endmsg
end
arch rod1
level 0
end
arch mob1
y 1
level 0
end
arch rod1
x 1
end
arch mob1
x 1
y 1
level 1
end
arch rod1
x 2
level 115
end
arch mob1
x 2
y 1
level 200
end
arch rod1
x 3
level 116
end
arch mob1
x 3
y 1
level 201
end
arch mob1
x 5
arch rod1
level 0
end
end
arch mob1
x 5
y 1
arch mob1
level 0
end
end
