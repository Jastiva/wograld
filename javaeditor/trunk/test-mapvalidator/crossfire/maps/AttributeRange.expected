[0|0] Suspicious attribute value [rod1] [rod1] [casting level, 0]
[3|0] Suspicious attribute value [rod1] [rod1] [casting level, 116]
[5|0] Suspicious attribute value [rod1] [rod1] [casting level, 0]
[0|1] Suspicious attribute value [mob1] [mob1] [level, 0]
[3|1] Suspicious attribute value [mob1] [mob1] [level, 201]
[5|1] Suspicious attribute value [mob1] [mob1] [level, 0]
[5|1] Inventory environment [mob1] [mob1] [Monster & NPC, 0]
